core/extensions.js

```diff
@@ -23,6 +23,7 @@ goog.module.declareLegacyNamespace();
 /* eslint-disable-next-line no-unused-vars */
 const Block = goog.requireType('Blockly.Block');
 const utils = goog.require('Blockly.utils');
+goog.requireType('Blockly.Mutator');
```

Comment:
In a `goog.module` module, a `goog.requireType` should always be assigned to a `const` alias and that alias used consistently to refer to it in type annotations.  Failure to do this causes compiler errors in strict mode.